soundbay/models.py

```diff
@@ -364,3 +364,26 @@ def __init__(self, num_classes=2, pretrained=True):
     def forward(self, x):
         x = x.repeat(1, 3, 1, 1)
         return self.resnet(x)
+
+class EfficientNet2D(nn.Module):
+    """
+    EfficientNet model for 3 channel ("RGB") input.
+    """
+    def __init__(self, num_classes=2, pretrained=True):
+        super(EfficientNet2D, self).__init__()
+        
+        # Load a pre-trained EfficientNet-B7 model from torchvision
+        self.efficientnet = models.efficientnet_b7(pretrained=pretrained)
```

Comment:
The `pretrained` parameter is deprecated.
It should be called like the resnet:
`resnet = models.resnet18(weights=ResNet18_Weights.DEFAULT) if pretrained else models.resnet18(weights=None)`